UICatalog/KeyBindingsDialog.cs

```diff
@@ -209,7 +209,7 @@ private void RecordView (View view)
             // (and always was wrong). Parents don't get to be told when new views are added
             // to them
 
-            view.Added += (s, e) => RecordView (e.Child);
+            view.Added += (s, e) => RecordView (e.SubView);
```

Comment:
Another anonymous subscription.

Reply:
ScrollBarView.cs will be removed later, so isn't a problem for now.

Reply:
That file belongs to `UICatalog` and objects are always discarded on exit. On the other hand, it is not part of this PR so it could be used in a new issue to prevent all these anonymous methods from unsubscribing.